components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -636,8 +636,6 @@ void RRTMGPRadiation::initialize_impl(const RunType /* run_type */) {
   m_n2vmr      = m_params.get<double>("n2vmr", 0.7906);
   m_covmr      = m_params.get<double>("covmr", 1.0e-7);
 
-  const double multiplier = m_params.get<double>("pool_size_multiplier", 1.0);
```

Comment:
Only used if RRTMGP_ENABLE_KOKKOS is defined, so move in that code block to fix warning.